supplier_app/views.py

```diff
@@ -110,7 +110,8 @@ def get_context_data(self, **kwargs):
         return context
 
     def form_valid(self, form):
-        if Company.objects.filter(name=form.data['name']).exists():
+        lowercase_company_names = [name[0].lower() for name in Company.objects.values_list('name')]
```

Comment:
You can use the flag `flat=True` in the method `value_list` to avoid doing  `name[0]`